### ISSUE TITLE
label_replace function fails with instant query

### ISSUE THREAD

**Issue by anakaiti** [grafana/loki#6447]
**Describe the bug**
Creating instant queries with label_replace function fails with the following error:

```
unexpected expr type (*syntax.LabelReplaceExpr) for ASTMapper type (logql.RangeMapper)
```

Running as range query with time range with 0 length outputs the expected result.

**To Reproduce**
1. Run query containing label_replace function as instant query. ex:

```
label_replace(
  sum by(gql_method) (
    count_over_time(
    {container_name="api-gateway"} 
    |= "dart"
    |= "\"logging.googleapis.com/labels\":{\"event_type\":\"response\"}"
    | regexp "\"query\":\"(?P<gql_method>[^()}]+)"
    [1m]
    )
  )
,"data", "count", "", "")
```

**Expected behavior**
Creating instant queries with label_replace function should work. 

**Environment:**
 - Grafana Cloud

**Screenshots, Promtail config, or terminal output**
If applicable, add any output to help explain your problem.

![Screenshot 2022-06-21T16 16 23Z](https://user-images.githubusercontent.com/5457202/174848392-8c8786ab-cb62-4b33-959e-3552299c9a15.png)

**Comment by dannykopping** [comment]
@ssncferreira I believe this was introduced in this PR:
https://github.com/grafana/loki/pull/5662/files#diff-57356c835885dd99f019a746d1fbd5ed78fade26de38fbc977fdc5466e59a1f1R90

**Comment by ssncferreira** [comment]
@anakaiti this bug fix is already merged to main: https://github.com/grafana/loki/pull/6515
And it will be released in the next Loki 2.6 release
